### PR TITLE
UN-539: Hero image on the Topic + Time Period explorer pages

### PR DESCRIPTION
**Pull request by jamesbiggs**
Ticket URL: https://national-archives.atlassian.net/browse/UN-539

## About these changes

Added `hero_image` field to the `Topic` and `TimePeriod` explorer pages.

## How to check these changes

Edit a `Topic` or `TimePeriod` explorer page, e.g. British State and Citizens, and add a `hero_image`. This should then display at the top banner of the page.

## Before assigning to reviewer, please make sure you have

- [x] Checked things thoroughly before handing over to reviewer.
- [x] Checked PR title starts with ticket number as per project conventions to help us keep track of changes.
- [x] Ensured that PR includes only commits relevant to the ticket.
- [x] Waited for all CI jobs to pass before requesting a review.
- [x] Added/updated tests and documentation where relevant.

## Merging PR guidance

Follow [docs\developer-guide\contributing.md](https://nationalarchives.github.io/ds-wagtail/developer-guide/contributing/)

## Deployment guidance

Follow [docs\infra\environments.md](https://nationalarchives.github.io/ds-wagtail/infra/environments/)

### CODE REVIEW

File: etna/core/models/forms.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+__all__ = ["RequiredHeroImagePageForm"]
+
+
+class RequiredHeroImagePageForm(WagtailAdminPageForm):
+    """
+    Overriding the base form class on select pages to allow for 'required' decorative
+    images that do not impact the function of the page. This allows for a non-required field
+    in the database, but a required field in the front-end, so that the hero_image cannot be
+    blank. This satisfies test cases as the hero_image doesn't need to be set.
+    """
```
Comment (ababic) [inline etna/core/models/forms.py:12]: 🥇

File: etna/collections/models.py [hunk 2]
```diff
@@ -288,9 +314,23 @@ class TimePeriodExplorerPage(AlertMixin, BasePageWithIntro):
...
+        get_image_model_string(),
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name="+",
+    )
     start_year = models.IntegerField(blank=False)
     end_year = models.IntegerField(blank=False)
     content_panels = BasePageWithIntro.content_panels + [
+        MultiFieldPanel(
```
Comment (ababic) [inline etna/collections/models.py:327]: Why not just use a single `FieldPanel` here?

File: etna/collections/models.py [hunk 3]
```diff
@@ -288,9 +314,23 @@ class TimePeriodExplorerPage(AlertMixin, BasePageWithIntro):
...
+            heading="Hero image",
+            classname="collapsible",
```
Comment (ababic) [inline etna/collections/models.py:329]: I *THINK* all panels might be collapsible by default in the latest Wagtail

File: etna/collections/models.py (hunk 2 above)
Comment (jamesbiggs) [reply etna/collections/models.py:327]: Ah, sorry Andy! I meant to put some context in this PR - we're going to add a "hero image caption" at some point once it's been spec'd out. I thought I'd just put this here to get it started, if that's okay? I did it for the last hero image addition too

**Comment by jamesbiggs** [comment]
Closing this PR, will resolve this ticket in another PR :)

**Pull request closed without merging**